### DISCUSSION TITLE
About cross-code location run status sensors

### DISCUSSION THREAD

**Discussion by raiAmagi** [dagster-io/dagster#23001] [Q&A]
I am reading and studying the documentation to use Dagster.
I am reading the sensor documentation now.

Please tell me about the following two points

1. how do I specify the code_location to be monitored when using run_failure_sensor?

　I understand that there are two ways to specify the code_location: location_name and repository_name.

　What is the difference between them?

　There is no explanation in the API document of Class "JobSelector".

It says that repository has replaced Definitions since Ver. 1.1.6. Is repository_name also a deprecated option?

2. about the sample PG
　In the sample PG, location_name="defs" but
　It says "if the job with code_location_a succeeds".

　Is this simply a misdescription of location_name="code_location_a"?

**Comment by garethbrickman** [comment]
I see you're referring to this [cross-code location run status sensors documentation](https://docs.dagster.io/concepts/partitions-schedules-sensors/sensors#cross-code-location-run-status-sensors). 

1. **location_name** refers to the name of the code location where the jobs are defined. **repository_name**: refers to the name of the repository within a code location. It's likely you won't have a repository and your code location will just be a single name. If you did have a repository, you would see your code location's name structured in two parts like `foo@bar`, representing `{repository}@{location}`

2. Yes I think in the example it's been misidentified.

**Reply by babaMar** [reply]
I don't see the example in the documentation, I'm looking into a similar use case where I need to monitor some jobs crated in code location A, to trigger a job created in code location B.

**Reply by gsparre** [reply]
@babaMar [legacy-docs](https://legacy-docs.dagster.io/concepts/partitions-schedules-sensors/sensors#cross-code-location-run-status-sensors)

**Comment by raiAmagi** [comment]
Thank you for your response.

I now understand how to use location_name and repository_name.

I will try it as soon as possible.

**Reply by babaMar** [reply]
did you manage to find a way?
